skyportal/tests/frontend/test_allocation.py

```diff
@@ -0,0 +1,108 @@
+from selenium.webdriver.common.by import By
+from skyportal.tests import api
+
+
+def one_request_comment_process(
```

Comment:
Nice! Really appreciate you creating a method to shorten the code

Reply:
Can you just add a dosctring for it? Otherwise it doesn't really speak for itself, hard to understand what it's for at a glance

Reply:
☑ Done